packages/os/etc-systemd-network.mount

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=systemd-networkd configuration directory
+DefaultDependencies=no
+Conflicts=umount.target
+Before=generate-network-config.service
+After=selinux-policy-files.service
+Wants=selinux-policy-files.service
+
+[Mount]
+What=tmpfs
+Where=/etc/systemd/network
+Type=tmpfs
+Options=nosuid,nodev,noexec,noatime,context=system_u:object_r:lease_t:s0,mode=0755
+
+[Install]
+RequiredBy=systemd-networkd.service
```

Comment:
A judgment call but I prefer "WantedBy" unless the other service can't possibly work without this.

```suggestion
WantedBy=systemd-networkd.service
```